root/static/scripts/account/components/ApplicationForm.js

```diff
@@ -93,7 +93,7 @@ class ApplicationForm extends React.Component<Props, State> {
         />
         <FormRowURLLong
           field={this.state.form.field.oauth_redirect_uri}
-          label={addColonText(l('Callback URL'))}
+          label={addColonText(l('Callback URI'))}
```

Comment:
Not sure what this unifies with, we don't seem to have a `Callback URI` string at the moment (you mean the longer doc string below?) but it seems sensible, anyway.

Reply:
Yes, it unifies with https://github.com/metabrainz/musicbrainz-server/blob/73c126c3ccf85b571f735e00a0fead613482bbd1/root/static/scripts/account/components/ApplicationForm.js#L104

It is technically both an URI and an URL, but we use URI in the code, and this is a tech field anyway.